concent_api/verifier/tasks.py

```diff
@@ -144,27 +152,126 @@ def blender_verification_order(
                 subtask_id,
                 VerificationResult.ERROR,
                 str(completed_process.stderr),
-                'verifier.blender_verification_order.running_blender'
+                ErrorCode.VERIFIER_RUNNING_BLENDER_FAILED.name
             )
             return
-    except SubprocessError as e:
+    except SubprocessError as exception:
         verification_result.delay(
             subtask_id,
             VerificationResult.ERROR,
-            str(e),
-            'verifier.blender_verification_order.running_blender'
+            str(exception),
+            ErrorCode.VERIFIER_RUNNING_BLENDER_FAILED.name
         )
         return
 
     # Verifier deletes source files of the Blender project from its storage.
     # At this point there must be source files in VERIFIER_STORAGE_PATH otherwise verification should fail before.
     source_files_list = get_files_list_from_archive(
-        os.path.join(settings.VERIFIER_STORAGE_PATH, source_package_path)
+        generate_verifier_storage_file_path(source_package_path)
     )
     for file_path in source_files_list + [source_package_path]:
         delete_file(file_path)
 
-    verification_result.delay(
-        subtask_id,
-        VerificationResult.MATCH.name,
-    )
+    blender_output_file_name = generate_blender_output_file_name(scene_file)
+    upload_file_name = generate_upload_file_name(subtask_id, output_format)
+
+    # Read Blender output file.
+    try:
+        with open(generate_verifier_storage_file_path(blender_output_file_name), 'r') as upload_file:
+            upload_file_content = upload_file.read()
+    except OSError as exception:
```

Comment:
Ok, however the issue description says that we should continue.

Reply:
Like I said elsewhere, this case is for uploading the rendering result for audit purposes. If it fails, well, tough luck. We should not stop verification because of that.